services/slam/fake/slam_test.go

```diff
@@ -34,6 +39,31 @@ func TestFakeSLAMPosition(t *testing.T) {
 	test.That(t, pInFrame2, test.ShouldResemble, pInFrame2)
 }
 
+func TestFakeSLAMGetPosition(t *testing.T) {
+	expectedComponentReference := ""
+	slamSvc := &SLAM{Name: "test", logger: golog.NewTestLogger(t)}
+
+	p, componentReference, err := slamSvc.GetPosition(context.Background(), slamSvc.Name)
+	test.That(t, err, test.ShouldBeNil)
+	test.That(t, componentReference, test.ShouldEqual, expectedComponentReference)
+
+	// test.ShouldBeBetween is used here as tiny differences were observed
+	// in floating point values between M1 mac & arm64 linux which
+	// were causing tests to pass on M1 mac but fail on ci.
+
+	test.That(t, p.Point().X, test.ShouldBeBetween, -0.005666600784453, -0.005666600784454)
```

Comment:
[nit] can these numbers be converted to a single point and a variable for a delta? in case they need to be adjusted again